eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -30,6 +31,10 @@ parameters:
                 template: %ezplatform.default_view_templates.content.line%
                 match: []
         embed:
+            image:
+                template: '%ezplatform.default_view_templates.content.embed_image%'
+                match:
+                    IsEmbedImage: true
```

Comment:
so this comes from "ez-embed-type-image"? Is this mapped directly on form the content type? Seems a bit hardcoded, would be more straight forward if match type was against content type here, but that's not possible maybe?